app/controllers/users/passwords_controller.rb

```diff
@@ -28,5 +30,20 @@ def user
       @user ||= User.find_by(email: email)
     end
 
+    def check_reset_timestamp(user)
```

Comment:
@eyss1234 can you please refactor this method? It says check reset timestamp but you are updating a value there too. plus the if else with return just false seems little odd :) ping me on slack if any questions